website/package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "next dev",
     "build": "next build",
     "build:analyze": "cross-env ANALYZE=true next build",
-    "start": "next start",
+    "start": "node .next/standalone/server.js",
```

Comment:
Nextjs warn when run next start with standalone mode. I though this will fix the CI but no luck